pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -252,7 +252,9 @@ func (r *ReconcileKnativeServing) checkDeployments(instance *servingv1alpha1.Kna
 			if err := r.client.Get(context.TODO(), key, deployment); err != nil {
 				instance.Status.MarkDeploymentsNotReady()
 				if errors.IsNotFound(err) {
-					return nil
+					// maybe deployments deleted manually
+					log.Info("checkDeployments", u.GetName(), " not found, maybe deleted manually")
+					instance.Status.MarkInstallFailed("deployments missed")
```

Comment:
@yu2003w I have resolved this issue in the PR 83 for the refactoring, by re-implementing this fuction..

Reply:
I don't think this change is necessary. The `ApplyAll` will fix it anyway, will it?

Reply:
This code path will most often be taken in the time it takes kubernetes to spin up the deployments after the operator initially creates them from its manifest. For that pretty common race condition, I think this change may add more confusion than clarity. So it's not clear to me exactly what problem this is solving.

Reply:
I installed kserving with operator by override registry images which resides in private docker registry. Because lack of image pull secretes, deployments are not ready when deployed.
Then I deleted two deployments webhook and controller.
Later I created registry secret and patched to proper service account. And deployment web hook and controller are not recreated.
So I want to fix this issue.
As the problem has been resolved in PR#83, I will close this PR.